config_defaults.yml

```diff
@@ -84,6 +84,8 @@
   branches:
     - master
     - /^v\d/
+  use_litmus:
+    enabled: false
```

Comment:
Do we envision a future where `use_litmus` has additional sub-options? In `appveyor.yml` defaults, it's just a simple boolean: `use_litmus: false` so it would be nice to be consistent with that unless there is a reason not to be.